Ryujinx.HLE/HOS/Services/Friend/ServiceCreator/IFriendService.cs

```diff
@@ -120,6 +120,45 @@ public ResultCode GetFriendList(ServiceCtx context)
             return ResultCode.Success;
         }
 
+        [CommandHipc(10120)] // 10.0.0+
+        // nn::friends::IsFriendListCacheAvailable(nn::account::Uid userId) -> bool
+        public ResultCode IsFriendListCacheAvailable(ServiceCtx context)
+        {
+            UserId userId = context.RequestData.ReadStruct<UserId>();
+
+            if (userId.IsNull)
+            {
+                return ResultCode.InvalidArgument;
+            }
+
+            // TODO: Service mount the friends:/ system savedata and try to load friend.cache file, returns true if exists, false instead. 
+            // NOTE: If no cache is available, guest then calls nn::friends::EnsureFriendListAvailable, we can avoid that by fake the cache check.
+            context.ResponseData.Write(true);
```

Comment:
```suggestion
            context.ResponseData.Write(1);
```
The bool is probably padded to 4 bytes?

Reply:
After a double check of the RE, it's not!